src/models/code2vec.py

```diff
@@ -0,0 +1,86 @@
+from modelforge import register_model, Model, split_strings, merge_strings
+from collections import defaultdict
+
+
+@register_model
+class Code2Vec(Model):
```

Comment:
*Code2VecFeatures , we will proabably have a second Model for the trained ML model